app/components/TipsDetails.tsx

```diff
@@ -20,6 +21,7 @@ const screenHeight = Dimensions.get('window').height;
 const screenWidth = Dimensions.get('window').width;
 
 export const TipsDetails = ({ schoolName, canEdit }: { schoolName: string; canEdit?: boolean }) => {
+  const { userInfo } = useContext(Context); // Assuming userInfo contains isAdmin property
```

Comment:
Don't think we need this comment — if isAdmin doesn't exist then the boolean check will just return false. We only add this field to the admin (val)